### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by alpesh12345**
It's an HarmonyOS library that provides an Animated striped button to show the loading process.

### CODE REVIEW

File: entry/src/main/java/com/github/nikartm/stripedprocessbutton/slice/MainAbilitySlice.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package com.github.nikartm.stripedprocessbutton.slice;
+
+import com.github.nikartm.stripedprocessbutton.ResourceTable;
```
Comment (samuel-abhishek) [inline entry/src/main/java/com/github/nikartm/stripedprocessbutton/slice/MainAbilitySlice.java]: sort the imports : project imports need to ohos imports

File: entry/src/main/resources/base/layout/ability_main.xml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        ohos:text_color="#FFF8F6F6"
+        app:spb_stripeRevert="true"
+        app:spb_background="#6D4C41"
+        app:spb_mainStripColor="#6D4C41"
+        app:spb_subStripeColor="#FF9800"
+        app:spb_stripeTilt="15"
+        app:spb_stripeAlpha="0.8"
+        app:spb_loadingText="Loading..."
+        app:spb_cornerRadius="270" />
+
```
Comment (samuel-abhishek) [inline entry/src/main/resources/base/layout/ability_main.xml:75]: Remove the unwanted spaces. Applies to all the files

**Review by samuel-abhishek — COMMENTED** [review]
Please do the mentioned changes

File: entry/src/main/java/com/github/nikartm/stripedprocessbutton/slice/MainAbilitySlice.java (hunk 1 above)
Comment (alpesh12345) [reply entry/src/main/java/com/github/nikartm/stripedprocessbutton/slice/MainAbilitySlice.java]: done

File: entry/src/main/resources/base/layout/ability_main.xml (hunk 2 above)
Comment (alpesh12345) [reply entry/src/main/resources/base/layout/ability_main.xml:75]: done

**Review by samuel-abhishek — APPROVED** [review]
LGTM

**Pull request merged**